src/web/views/enter-code.njk

```diff
@@ -33,6 +33,6 @@
     {% endcall %}
 
     {# TODO DW HTBHF-1712 update link to send new confirmationCode  #}
-    <p class="govuk-body-m"><a href="#">{{ requestANewCode }}</a> {{ ifItDoesNotArrive }}</p>
+    <p class="govuk-body-m"><a href="/send-code">{{ requestANewCode }}</a> {{ ifItDoesNotArrive }}</p>
```

Comment:
Can we add an id to this link so we can use `findById` in `Page`? We then don't need the new method `findByLinkText`.

Reply:
Added id.